examples/zephyr/fw_update/pytest/test_sample.py

```diff
@@ -0,0 +1,107 @@
+from golioth import Client
+import logging
+import pytest
+import time
+import yaml
+import datetime
+import os
+
+LOGGER = logging.getLogger(__name__)
+
+UPDATE_VERSION = '255.8.9'
+UPDATE_PACKAGE = 'main'
+API_URL = "https://api.golioth.io"
```

Comment:
FYI this targets prod (so the workflow needs to also use the prod API key secret)